examples/src/main/java/com/hedera/hashgraph/sdk/examples/advanced/DeleteFile.java

```diff
@@ -39,12 +39,10 @@ public static void main(String[] args) throws HederaException {
         // you can easily use the bytes of a file instead.
         byte[] fileContents = "Hedera hashgraph is great!".getBytes();
 
-        TransactionId txId = new FileCreateTransaction().setExpirationTime(
-            Instant.now()
-                .plus(Duration.ofSeconds(2592000)))
-            // Use the same key as the operator to "own" this file
+        TransactionId txId = new FileCreateTransaction()
             .addKey(OPERATOR_KEY.publicKey)
             .setContents(fileContents)
+            .setMaxTransactionFee(200_000_000)
```

Comment:
We'd like to push people to use the new `Hbar` class, so this would be `Hbar.of(2)`.

Reply:
Sweet - updated!